inflight.js

```diff
@@ -13,6 +13,7 @@ function _inflight (unique, doFly) {
       return resolve(doFly())
     }))
     active[unique].then(cleanup, cleanup)
+    function cleanup() { delete active[unique] }
```

Comment:
The other copy of `cleanup()` should also be removed, but really I'd prefer that you just remove this commit and 1eb2b41a157ee16166392e82e058e87ea64d3b86